packaging/package.yml/en.md

```diff
@@ -98,7 +101,9 @@ Optimize Value | Description
 **no-bind-now** | Configure the package to disable certain flags, where RELRO is unsupported.
 **no-symbolic** | Disable `-Wl,-Bsymbolic-functions` linker flag.
 **unroll-loops** | Enable `-funroll-loops`. Use this sparingly, only when it provides proven benefit.
-**thin-lto** | Enable Thin Link Time Optimization `-flto=thin` with the `ld.gold` linker.
+**runpath** | Enable -Wl,--enable-new-dtags to make linker use RUNPATH's instead of RPATH's.
```

Comment:
Let's get -Wl,--enable-new-dtags set with inline code (`) to be consistent with everything else.